Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -137,20 +137,13 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
        .. code-block:: shell-session
 
-          minikube start --network-plugin=cni --cni=false
+          minikube start --cni=cilium
 
        .. note::
 
-          From minikube v1.12.1+, cilium networking plugin can be enabled directly with
```

Comment:
Would it make sense to flip this so that we'd say in the note that for v1.12.0 and earlier the option is `--network-plugin-cni --cni=false` and Cilium is installed manually? Or is that already so ancient that it's not worth mentioning?

Reply:
I can make this change if needed. IMHO it comes down to whether docs should be provided for an unsupported minikube version.

Reply:
@joamaki do you have an opinion on my ^ comment?

Reply:
I think it makes sense to leave the old version out. Eventually, we'd have to remove the old instructions anyway. If users need to see the old instructions, they can go back in time to view the old installation instructions using the version selector in the docs.

Reply:
I think it makes sense to leave the old version out. Eventually, we'd have to remove the old instructions anyway. If users need to see the old instructions, they can go back in time to view the old installation instructions using the version selector in the docs.

Reply:
^ is exactly what I was thinking.

Reply:
holding off until this discussion is resolved, @joamaki please ping / re-label when you're good :)

Reply:
@joamaki when you have a moment, do you mind commenting to unblock this PR?

Reply:
Makes sense to me!